webextensions/manifest/commands.json

```
@@ -21,6 +21,10 @@
             "safari": {
               "version_added": "14",
               "notes": "Ability to change the keyboard shortcut for a command not supported."
```

We could probably update the wording of this note to flow better:

```suggestion
              "notes": "Changing the keyboard shortcut for a command is not supported."
```